examples/stream_record_task_split.py

```diff
@@ -0,0 +1,92 @@
+"""
+====================
+stream record and split mock streams
+====================
+Use :func:`~neurobooth_os.iout.mock_device_streamer` to generate mock streams and liesl to record.
+
+Only works in Windows or Linux using Wine
```

Comment:
Liesl now works on Mac :) I still need to add the CI for Mac on Liesl but it will be done soon